app/views/hyrax/base/_currently_shared.html.erb

```diff
@@ -0,0 +1,66 @@
+<%#
+# form object.class = SimpleForm::FormBuilder
+#   For works (i.e. GenericWork):
+#   - form object.object = Hyrax::GenericWorkForm
+#   - form object.object.model = GenericWork
+#   - use the work itself
+#   For file_sets:
+#   - form object.object.class = FileSet
+#   - use work the file_set is in
+#   No other object types are supported by this view. %>
+<% if f.object.respond_to?(:model) && f.object.model.work?
+  object_acting_upon = f.object
+elsif f.object.file_set?
+  object_acting_upon = f.object.in_works.first
```

Comment:
@laritakr This is a serious shot in the dark, but what should the fallback be? (If any?)

I'm okay with a `raise`

Reply:
I don't really remember, but I believe this partial is only used for works and file sets, so anything else going through here would be unexpected behavior. I say raise an exception.